internal/userutils/userutils_test.go

```diff
@@ -0,0 +1,66 @@
+package userutils_test
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/authd/internal/userutils"
+)
+
+//go:generate go build -o testhelpers/gpasswd ./testhelpers/gpasswd.go
```

Comment:
Better to build tools at testing phase, as generate is mostly to generate sources, and if not we should maybe add tags